ONSdigital/dp-identity-api
# Feature/update login endpoint with cognito

### What

- Call InititateAuth endpoint in Cognito
- Return 201 with tokens in the header and expiration time in the body if authorised
- Return 401 if not authorised
- Return 403 if too many attempts
- Return 500 if internal server error 
- Return 400 for other client errors from cognito

- Add unit tests
- Add feature tests and cognito stub
### How to review

- make test
- make test-component
- Run make debug and a post request can be made to /tokens with a user to see the different scenarios. Please let me know fi you need user details

### Who can review

Anyone but @AWitcherONS and myself
